### DISCUSSION TITLE
core手动上传最新版成功，node也正常，还是显示“UnblockNeteaseMusic 未运行”，不能正常工作

### DISCUSSION THREAD

**Discussion by mysdarking** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#180] [Q&A]
<img src="/mysdarking/awesome-python-cn/blob/master/1.png?raw=true" alt="1.png">
<img src="/mysdarking/awesome-python-cn/blob/master/2.png?raw=true" alt="2.png">
<img src="/mysdarking/awesome-python-cn/blob/master/3.png?raw=true" alt="3.png">

Launching luci-app-unblockneteasmusic Debugging Tool...

OpenWrt info:
{
	"kernel": "5.15.58",
	"hostname": "OpenWrt",
	"system": "Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz",
	"model": "VMware, Inc. VMware7,1",
	"board_name": "vmware-inc-vmware7-1",
	"rootfs_type": "squashfs",
	"release": {
		"distribution": "1004",
		"version": "22.03.0-rc6",
		"revision": "08.06.2022",
		"target": "x86/64",
		"description": "OpenWrt 08.06.2022 by Kiddin'"
	}
}

uclient-fetch info:
Package: uclient-fetch
Version: 2021-05-14-6a6011df-1
Depends: libc, libuclient20201210
Provides: wget
Status: install user installed
Section: net
Architecture: x86_64
Size: 7944
Filename: uclient-fetch_2021-05-14-6a6011df-1_x86_64.ipk
Description: Tiny wget replacement using libuclient
Installed-Time: 1659321992

Package: libustream-wolfssl20201210
Version: 2022-01-16-868fd881-1
Depends: libc, libubox20220515, libwolfssl5.4.0.ee39414e
Provides: libustream-wolfssl
Conflicts: libustream-openssl
Status: install user installed
Section: libs
Architecture: x86_64
Size: 5019
Filename: libustream-wolfssl20201210_2022-01-16-868fd881-1_x86_64.ipk
Description: ustream SSL Library (wolfssl)
Installed-Time: 1659321992

Package: libustream-openssl20201210
Version: 2022-01-16-868fd881-1
Depends: libc, libubox20220515, libopenssl1.1
Provides: libustream-openssl
Status: unknown ok not-installed
Section: libs
Architecture: x86_64
Size: 5530
Filename: libustream-openssl20201210_2022-01-16-868fd881-1_x86_64.ipk
Description: ustream SSL Library (openssl)

Package: libustream-mbedtls20201210
Version: 2022-01-16-868fd881-1
Depends: libc, libubox20220515, libmbedtls12
Provides: libustream-mbedtls
Conflicts: libustream-openssl, libustream-wolfssl
Status: unknown ok not-installed
Section: libs
Architecture: x86_64
Size: 5341
Filename: libustream-mbedtls20201210_2022-01-16-868fd881-1_x86_64.ipk
Description: ustream SSL Library (mbedtls)

Downloading 'https://api.github.com/repos/UnblockNeteaseMusic/server/commits?sha=enhanced&path=precompiled'
Connecting to 20.205.243.168:443
Writing to stdout

-                    100% |*******************************|   122k  0:00:00 ETA
Download completed (125176 bytes)
ec282cb17665e09ac6f25dfe21f10928db1ca8d6

Node.js info:
Package: node
Version: v16.16.0-1
Depends: libc, libstdcpp6, libopenssl1.1, zlib, libnghttp2-14, libuv1, libcares, libatomic1
Status: install ok installed
Section: lang
Architecture: x86_64
Size: 11509275
Filename: node_v16.16.0-1_x86_64.ipk
Description: Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world. *** The following preparations must be made on the host side. *** 1. gcc 8.3 or higher is required. 2. To build a 32-bit target, gcc-multilib, g++-multilib are required. 3. Requires libatomic package. (If necessary, install the 32-bit library at the same time.) ex) sudo apt-get install gcc-multilib g++-multilib
Installed-Time: 1659978435

Node.js is placed at /usr/bin/node
Node.js version: v16.16.0

luci-app-unblockneteasmusic info:
Package: luci-app-unblockneteasemusic
Version: git-22.215.56026-deef048
Depends: libc, busybox, dnsmasq-full, ipset, jsonfilter, node, UnblockNeteaseMusic, uclient-fetch, ucode
Status: install user installed
Section: luci
Architecture: all
Size: 13836
Filename: luci-app-unblockneteasemusic_git-22.215.56026-deef048_all.ipk
Conffiles:
 /etc/config/unblockneteasemusic e9413f9ce065cfb7b875d1948c6d3258772ab8854b7ccf193df49c3c8b6cca20
Description: LuCI support for UnblockNeteaseMusic
Installed-Time: 1659978513

-rw-r--r--    1 root     root         288 Aug  9 01:15 /etc/config/unblockneteasemusic
-rwxr-xr-x    1 root     root        8.3K Aug  3 23:33 /etc/init.d/unblockneteasemusic

/usr/share/unblockneteasemusic:
drwx------    2 root     root        3.4K Aug  9 01:12 core
-rw-r--r--    1 root     root          41 Aug  9 01:12 core_local_ver
-rwxr-xr-x    1 root     root        5.2K Aug  3 23:33 debugging.sh
-rwxr-xr-x    1 root     root         348 Aug  3 23:33 log_check.sh
drwxr-xr-x    2 root     root        3.4K Aug  9 01:08 rules
-rwxr-xr-x    1 root     root        3.2K Aug  3 23:33 update.sh

config unblockneteasemusic 'config'
	option use_custom_cookie '0'
	option auto_update '1'
	option update_time '3'
	option advanced_mode '0'
	option local_vip '1'
	option enable_flac '1'
	option replace_music_source 'lower_than_320kbps'
	option enable '1'
	option music_source 'default'

UnblockNeteaseMusic Node.js info:
Git HEAD version: ec282cb17665e09ac6f25dfe21f10928db1ca8d6
Core version: Not Found
-rw-r--r--    1 root     root        4.2M Aug  9 01:12 app.js
-rw-r--r--    1 root     root        3.6M Aug  9 01:12 bridge.js
-rw-r--r--    1 root     root      229.7K Aug  9 01:12 ca.crt
-rw-r--r--    1 root     root      223.8K Aug  9 01:12 server.crt
-rw-r--r--    1 root     root      222.1K Aug  9 01:12 server.key

Netease networking info:
< Location: http://m10.music.126.net/20220809014500/c0f3ed70f3a8d12d43cdf30379fdbf73/ymusic/obj/w5zDlMODwrDDiGjCn8Ky/3057940231/3d32/93a3/4db1/0757f3a378b41e09ca2767c1abd47afa.mp3
{"code":200,"data":{"music.163.com":{"ip":["115.236.118.34","115.236.121.4","59.111.19.33","59.111.160.195","182.92.170.253","118.24.63.156","193.112.159.225","39.105.63.80","47.100.127.239","39.105.175.128"],"ipv4ScoreDelay":0,"prefer":"ipv4","score":false,"ttl":600}}}

Port status:
No instance found on port 5200.
No instance found on port 5201.

PROCD running info:
{
	
}

Local version: NOT FOUND, latest version: ec282cb17665e09ac6f25dfe21f10928db1ca8d6.
Updating core...
Failed to download app.js.

Local version: ec282cb17665e09ac6f25dfe21f10928db1ca8d6, latest version: ec282cb17665e09ac6f25dfe21f10928db1ca8d6.
You're already using the latest version.

Log is available at:
https://litter.catbox.moe/vfktw1.txt
https://transfer.sh/hSRGWQ/unm-debugging-output.txt

**Comment by 1715173329** [comment]
1. 从 Actions 或 Releases 中下载最新的版本安装
2. 检查网络状态，重新下载。

**Reply by mysdarking** [reply]
删掉了/usr/share/下所有unblockneteasemusic的文件夹，还有etc/config/下unblockneteasemusic的文件，从系统-软件包-上传软件包最新的luci-app-unblockneteasemusic_3.0-3_javascript_all.ipk，安装成功，更新core成功，路由重启终于成功了。但最新2.10.*版本显示网络不通，2.5的可以

**Reply by 1715173329** [reply]
不要用新版，不支持，也没什么用。
